test/unit/core/2d_primitives.js

```diff
@@ -38,16 +50,28 @@ suite('2D Primitives', function() {
         },
         Error, 'got unwanted exception');
     });
+    test('wrong param type at #7', function() {
+      assert.doesNotThrow(function() {
+          myp5.arc(1, 1, 10.5, 10, 0, Math.PI, 'pie', 'string');
+        },
+        Error, 'got unwanted exception');
+    });
   });
 
   suite('p5.prototype.ellipse', function() {
     test('should be a function', function() {
       assert.ok(myp5.ellipse);
       assert.typeOf(myp5.ellipse, 'function');
     });
-    test('no friendly-err-msg', function() {
+    test('no friendly-err-msg, 2D', function() {
+      assert.doesNotThrow(function() {
+          myp5.ellipse(0, 0, 100, 100);
+        },
+        Error, 'got unwanted exception');
+    });
+    test('no friendly-err-msg, 3D', function() {
       assert.doesNotThrow(function() {
-          myp5.ellipse(0, 0, 100);
+          myp5.ellipse(0, 0, 100, 100, 24);
```

Comment:
I know some earlier comments were about GL test issues, but everything is passing for me. :)

Reply:
Sounds good. I see the same failures as before, but don't think they are related to my code.